### PR TITLE
Settings - Clarify logging for userconfig loading

### PR DESCRIPTION
**Pull request by PabstMirror**

### CODE REVIEW

**Comment by commy2** [comment]
Very good! Testing...

File: addons/settings/fnc_initDisplayMain.sqf [hunk 1]
```diff
@@ -29,8 +28,13 @@ if (_file != "") then {
     };
 
     if (_fileExists) then {
+        INFO_1("Userconfig: Loading file [%1]",_file);
         _userconfig = preprocessFile _file;
+    } else {
+        INFO_1("Userconfig: Could not open file [%1]",_file);
```
Comment (commy2) [inline addons/settings/fnc_initDisplayMain.sqf]: Works, but technically the file could also be just empty. Probably not worth mentioning it.

Comment (jonpas) [reply addons/settings/fnc_initDisplayMain.sqf]: `Could not open file [%1] (empty or <other reason>)`?

Comment (commy2) [reply addons/settings/fnc_initDisplayMain.sqf]: File not found or empty.
This also reminds me that the log comments should have periods at the end imo.

**Pull request merged**